repos/fdbt-site/src/pages/globalSettings.tsx

```diff
@@ -33,7 +33,7 @@ const GlobalSettings = ({ globalSettingsCounts, referer }: GlobalSettingsProps):
                     </div>
 
```

Comment:
Can we change the widths to be consistent?
To me it looks better if we make them all 3/4 instead of the other pages that are 2/3?